test/e2e/mv3/multiple-restarts.spec.js

```diff
@@ -257,11 +258,18 @@ describe('MV3 - Restart service worker multiple times', function () {
 
         await openDapp(driver);
 
+        await driver.getAllWindowHandles();
+        await driver.getWindowTitles();
+
         await clickSendButton(driver);
         await driver.waitUntilXWindowHandles(2);
 
         await switchToWindow(driver, WINDOW_TITLES.TestDApp);
         await terminateServiceWorker(driver);
+
+        await driver.getAllWindowHandles();
+        await driver.getWindowTitles();
```

Comment:
Can u explain a bit on these 2 lines ?
For example, I can see inside `waitUntilXWindowHandles`, we have duplicated calls  
```
    while (timeElapsed <= timeout) {
      windowHandles = await this.driver.getAllWindowHandles();
```